client/components/catalog/Card/index.vue

```diff
@@ -133,22 +146,25 @@ export default {
 .card-body {
   padding: 0.625rem 0 0;
 
-  .v-card__title {
-    line-height: 1.75rem;
+  .header {
+    display: flex;
+    align-items: center;
   }
 
   .schema-name {
-    padding: 0 0 0 0.25rem;
+    flex-grow: 1;
     color: #fafafa;
     font-size: 0.75rem;
     font-weight: 500;
     letter-spacing: 1px;
     text-transform: uppercase;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
Vuetify has a class for this called [text-truncate](https://vuetifyjs.com/en/styles/text-and-typography/#wrapping-and-overflow).